File: iteration-2/src/Checker.java
public abstract class Checker {
    private Student student;

    protected Checker(Student student) {this.student = student;}

    public Student getStudent() {return student;}

    public void setStudent(Student student) {this.student = student;}

    public Course raiseCourse(){return null;}

    public String checkerMessage(){return "Checker Message";}
}

File: iteration-3/Checker.py
class Checker: 
    def __init__(self, student):
        self.student = student

    def get_student(self):
        return self.student

    def set_student(self, student):
        self.student = student

    def raise_course(self):
        return None

    def raise_error(self):
        try:
            return "Abstract Error"
        except Exception as e:
            print(e)
